TELCODOCS-1826: Day 0 - Day 2 kmods (#75752)
openshift/openshift-docs: modules/kmm-layering-background.adoc

// Module included in the following assemblies:
//
// * updating/preparing_for_updates/kmm-preflight-validation.adoc

:_mod-docs-content-type: CONCEPT
[id="kmm-layering-background_{context}"]
= Layering background

> When a Day 0 kmod is installed in the cluster, it means that layering was applied through the Machine Config Operator (MCO) and {product-title} (OCP) upgrades won’t trigger node upgrades.

Reviewer: I think the guidance here is correct - no "OCP" needed in parentheses.

> When a Day 0 kmod is installed in the cluster, it means that layering was applied through the Machine Config Operator (MCO) and {product-title} (OCP) upgrades won’t trigger node upgrades.

Reviewer: ```suggestion
When a Day 0 kmod is installed in the cluster, layering is applied through the Machine Config Operator (MCO) and {product-title} (OCP) upgrades do not trigger node upgrades.
```

Reviewer: ^generally avoid past/future tense per [IBM Style](https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense) although if it needs to be past tense/future tense here for a specific reason, ignore this guidance.

Author: Changed.

> Unless you want to add new features to the driver, it will never need to be recompiled because the node’s OS will remain the same.

Author: changed.

> Unless you want to add new features to the driver, it will never need to be recompiled because the node’s operating system will remain the same.

Reviewer: ```suggestion
You only need to recompile the driver if you add new features to it, because the node’s operating system will remain the same.
```

Author: Changed.
